cmd/yggctl/main.go

```diff
@@ -131,6 +131,57 @@ func main() {
 			},
 			Action: dispatchAction,
 		},
+		{
+			Name:        "message-journal",
+			Usage:       "Display a list of all worker messages and emitted events",
+			UsageText:   "yggctl message-journal",
+			Description: "The message-journal command sends a dbus call to yggdrasil to retrieve a message journal containing all worker messages and emitted events.",
+			Flags: []cli.Flag{
+				&cli.BoolFlag{
+					Name:     "persistent",
+					Aliases:  []string{"p"},
+					Usage:    "Display worker messages and emitted events gathered across multiple sessions from persistent storage.",
+					Required: false,
+				},
+				&cli.UintFlag{
+					Name:     "truncate-message",
+					Aliases:  []string{"t"},
+					Usage:    "Truncates worker event messages if they exceed the specified character count.",
```

Comment:
Try to use the `BACKTICK` notation in Usage text if possible. It helps to
clarify the data types expected in the flag.